<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <rosparam command="load" file="$(find earth_rover_data)/config/config.yaml" />
  <node pkg="earth_rover_data" type="publisher_node.py" name="earth_rover_publisher"
        output="screen"
        clear_params="true"
        respawn="true"
        respawn_delay="30">
  </node>
</launch>